bears/general/KeywordBear.py

```diff
@@ -92,6 +97,9 @@ def run(self,
         '''
         comments = _get_comments(dependency_results)
 
+        if not comments:
+            return
```

Comment:
sorry but this means that if the language isn't known the bear won't even look for keywords